.github/workflows/test-unit-gh_runner.yml

```diff
@@ -54,10 +57,42 @@ jobs:
         run: |
           export LANG="en_US.UTF-8"
           sudo update-locale
-          sudo /opt/HPCCSystems/bin/unittests
+          while read test
+          do
+            echo "$test"
+            echo "--------------------------------------"
+            sudo /opt/HPCCSystems/bin/unittests -e "$test"
+            echo " "
+          done< <(/opt/HPCCSystems/bin/unittests -l | sort )
+
+      - name: Check for Core files
+        run: |
+          CORE_FILES=( $(sudo find /opt/HPCCSystems/bin/ /var/lib/HPCCSystems/ -iname 'core*' -type f -print) )
```

Comment:
are you sure these are the right directories to look in?

if unittest cores, it will dump in the current working director which will be the runner cwd afaics - i.e. there won't be any core files in /opt/HPCCSystems/bin/ or /var/lib/HPCCSystems/.

Also, is 'sudo' necessary?

If there can only be <=1 core and only from unittest in cwd then this could be simplified to:
```
    shopt -s nullglob
    CORE_FILES=(core*)
```

Reply:
Changed